xdsl/dialects/__init__.py

```diff
@@ -360,6 +365,7 @@ def get_transform():
         "onnx": get_onnx,
         "pdl": get_pdl,
         "printf": get_printf,
+        "opaque_ptr": get_opaque_ptr,
```

Comment:
? why not ptr? is it to avoid the confusion with MLIR for unknown operations in dialects?

Reply:
yep, exactly

Reply:
Hmm, I feel like we've had a discussion about this in the past, and I don't remember the conclusion. I would recommend naming this `xdsl_ptr` or `ptr_xdsl`, to make it clear what's happening.